hack/ci-deploy.sh

```diff
@@ -71,6 +71,11 @@ else
   kubectl_create -n=local-csi-driver -f="${SO_CSI_DRIVER_PATH}"
 fi
 
+if [[ -n "${SO_SCYLLACLUSTER_STORAGECLASS_NAME}" ]]; then
+  yq e --inplace ".spec.datacenter.racks[0].storage.storageClassName = \"${SO_SCYLLACLUSTER_STORAGECLASS_NAME}\"" "${DEPLOY_DIR}/manager/50_scyllacluster.yaml"
+elif [[ -n "${SO_SCYLLACLUSTER_STORAGECLASS_NAME+x}" ]]; then
+    yq e --inplace "del(.spec.datacenter.racks[0].storage.storageClassName)" "${DEPLOY_DIR}/manager/50_scyllacluster.yaml"
```

Comment:
```suggestion
    yq e --inplace 'del(.spec.datacenter.racks[0].storage.storageClassName)' "${DEPLOY_DIR}/manager/50_scyllacluster.yaml"
```
tiny nit, but can be coupled to the edit above

Reply:
I'd rather leave it as is, I think it's complicated enough already :sweat_smile:

Reply:
it's in line with the other ones but I don't mind